drivers/ph_oem/ph_oem.c

```diff
@@ -0,0 +1,486 @@
+/*
+ * Copyright (C) 2019 University of Applied Sciences Emden / Leer
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_ph_oem
+ * @{
+ *
+ * @file
+ * @brief       pH OEM device driver
+ *
+ * @author      Igor Knippenberg <igor.knippenberg@gmail.com>
+ * @}
+ */
+
+#include "xtimer.h"
+#include "assert.h"
+#include "periph/i2c.h"
+#include "periph/gpio.h"
+
+#include "ph_oem.h"
+#include "ph_oem_params.h"
+#include "ph_oem_regs.h"
+
+#define ENABLE_DEBUG    (0)
+#include "debug.h"
+
+#define I2C (dev->params.i2c)
```

Comment:
This clashes with a system define on both `cc2650-launchpad` and `cc2650stk`. Renaming the define should fix this.

Reply:
fixed 👍

Reply:
Turns out `I2C_DEV` is also already taken.
Try with `DEV_I2C`, that's what some other drivers do.

Reply:
Ups, my bad. Fixed now